### DISCUSSION TITLE
构建自己的信息简报 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#138] [Announcements]
# 构建自己的信息简报 · BMPI

使用RSS与Email打造自己的信息简报。

https://www.bmpi.dev/self/use-rss-email-read/

**Comment by xiaopiao009** [comment]
你好，我也想弄一个自己的信息简报，不知您这个是否考虑写一个小白教程？

**Comment by lionoggo** [comment]
想问下楼主，现在对应的流程是怎么样的？
